includes/admin/views/addons/addon-item.php

```diff
@@ -1,22 +1,26 @@
 <?php
 /**
- * Single Add-on Item View
- * Used on Add-Ons browser screen
+ * Single Add-on Item View.
+ * Used on Add-Ons browser screen.
  *
- * @since    3.22.0
- * @version  3.22.0
+ * @since 3.22.0
+ * @since [version] Updated image URLs to use the new local addon image directory with a fallback remote image.
+ * @version [version]
  */
 defined( 'ABSPATH' ) || exit;
-
 ?>
+
 <li class="llms-add-on-item type--<?php echo esc_attr( $addon->get( 'type' ) ); ?>" id="<?php echo esc_attr( $addon->get( 'id' ) ); ?>">
 
 	<div class="llms-add-on">
 
 		<a class="llms-add-on-link" href="<?php echo esc_url( $addon->get_permalink() ); ?>" target="_blank">
 
 			<header>
-				<img alt="<?php echo $addon->get( 'title' ); ?> Banner" src="<?php echo esc_url( $addon->get( 'image' ) ); ?>">
+				<object data="<?php echo esc_url( llms()->plugin_url() . '/assets/images/addons/' . basename( $addon->get( 'image' ) ) ); ?>">
+					<img alt="<?php echo $addon->get( 'title' ); ?> Banner" src="<?php echo esc_url( $addon->get( 'image' ) ); ?>">
+				</object>
```

Comment:
Have you considered instead something like checking if the file exists (or better readable) and use the image URL, otherwise fall back on the remote URL? Instead of introducing this object + img?

Reply:
Yeah, I did consider it, but thought `object + img` would be a better alternative. Anyways, `if` statement is much better.

Reply:
So you think the `if` would be better?

Reply:
Does the ternary statements look good? I don't think so. What do you think?

Reply:
to me it does, but I don't see it

Reply:
I prefer the ternary

Reply:
done 👍